mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```diff
@@ -5,14 +5,56 @@
 
     "$defs": {
         "tunable_param_meta": {
+            "description": "A dictionary of metadata about the tunable parameter.  Can be used by scripts for additional info when generating configs from the suggested values.",
             "type": "object",
             "additionalProperties": {
+                "$comment": "Only flat dictionaries are allowed.",
                 "type": ["array", "string", "boolean", "null", "number"],
                 "items": {
                     "type": ["string", "boolean", "null", "number"]
                 }
             }
         },
+        "numeric_range": {
+            "description": "Two element array representing the lower and upper bounds of the range.",
+            "type": "array",
+            "$comment": "items type left unspecified here",
+            "minItems": 2,
+            "maxItems": 2
+        },
+        "quantization": {
+            "description": "The number of buckets to quantize the range into.\nSee Also:\nhttps://automl.github.io/ConfigSpace/main/api/hyperparameters.html#module-ConfigSpace.api.types.float,\nhttps://automl.github.io/ConfigSpace/main/api/hyperparameters.html#module-ConfigSpace.api.types.integer",
+            "$comment": "type left unspecified here",
+            "minimum": 1
+        },
+        "log_scale": {
+            "description": "Whether to use log instead of linear scale for the range search.",
+            "type": "boolean"
+        },
+        "special_values": {
+            "description": "An array of values that may have special meaning for the target system and could be outside the usual search range.",
+            "type": "array",
+            "items": {
+                "description": "Some special values may have a different type than the numeric parameter (e.g., keyword \"AUTO\").",
+                "type": ["number", "string", "boolean", "null"]
```

Comment:
These expanded types were missing from a previous PR even though we added it in the code.
Should probably also add expanded tests for them too though.